cabal-install/tests/IntegrationTests.hs

```diff
@@ -0,0 +1,276 @@
+-- | Groups black-box tests of cabal-install and configures them to test
```

Comment:
I did check out `tasty-golden`, but it doesn't really seem to get us much since all the traversal stuff it pretty specialized anyway. Did you have something particular in mind?